bin.src/genDefectList.py

```diff
@@ -0,0 +1,265 @@
+#!/usr/bin/env python
+import argparse
+import lsst.afw.image as afwImage
+import lsst.afw.math as afwMath
+import lsst.daf.persistence as dafPersist
+from lsst.log import Log
+import numpy as np
+import os
+import pfs.drp.stella as drpStella
+try:
+    import lsst.afw.display as afwDisplay
+except ImportError:
+    afwDisplay = None
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument("--arm", help="Arm for which to create the defect list", type=str, default='r', choices=['b', 'n', 'm', 'r'])
+parser.add_argument("--badCols", help="List of bad columns delimited by ','", type=str, default="1023,1024,3071,3072")
```

Comment:
The bad columns are probably the same for all PFS CCDs, so I put them in the defaults.
Using ListField now for --badCols.